homeassistant/components/binary_sensor/insteon_plm.py

```diff
@@ -15,38 +15,43 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+SENSOR_TYPES = {'openClosedSensor': 'opening',
+                'motionSensor': 'motion',
+                'doorSensor': 'door',
+                'leakSensor': 'moisture'}
+
 
 @asyncio.coroutine
 def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     """Set up the INSTEON PLM device class for the hass platform."""
+    state_list = []
```

Comment:
I am good to call it an entity. the reason it is changed from a device is because in INSTEON terms it is not a device, it is a state within the device. It is a terminology mapping issue but I am fine to map to the HA terminology of entity. FYI, this was one of the major changes to the underlying insteonplm library. Many complex devices (such as FanLinc) have two entities inside them (i.e. a light at a fan in the FanLinc case) From an INSTEON perspective they are one device with two controllable entities (called states in that library). So calling it a state_list is mapping to the insteonplm naming convention but I am fine to map to the HA naming convention of entity.

Reply:
Good, please use entity convention inside home assistant.